test/fw/bin/pbs_swigify

```diff
@@ -305,9 +305,11 @@ if __name__ == '__main__':
         cmd = ['gcc', '-Wall', '-Wno-unused-variable', '-fPIC', '-shared',
                '-I' + pbsinclude]
         cmd += ['-I' + pythoninc]
-        cmd += ['pbs_ifl_wrap.c', pbs_conf['PBS_EXEC'] + '/lib/libpbs.a']
-        cmd += ['-lcrypto']
+        cmd += ['pbs_ifl_wrap.c', '-L' +
+                pbs_conf['PBS_EXEC'] + '/lib', '-lpbs']
         cmd += ['-o', '_pbs_ifl.so']
+        cmd += ['-L' + '/lib']
```

Comment:
I tried removing the library path and it's working as expected. I can remove it.